docs/source/dashboard.rst

```diff
@@ -0,0 +1,130 @@
+dashboard:
+
+.. figure:: images/dashboard_status.png
+    :alt: NEEDS ALT TEXT
+
+- Bytes stored: Cluster memory. 
+- Bytes per Worker: Memory per worker.
+- Task Processing/CPU Utilization/Occupancy: Tasks being processed by each worker/ NEEDS INFO/ NEEDS INFO
+- Task Stream: Individual task across threads.
+- Progress: Progress of a set of tasks.
+
+
+Bytes Stored and Bytes per Worker
+---------------------------------
+
+.. figure:: images/dashboard_memory.png
+    :alt: NEEDS ALT TEXT
+
+These two plots show a summary of the overall memory usage on the cluster,
+as well as the individual usage on each worker. In these plots, the colors and 
+shades have different meanings:
+    - Solid Blue: Managed memory under target (default 70% of memory)
+    - Solid Orange: Managed memory is close to the spilling target (default 70% of memory)
+
+For both blue and orange we have two more shades:
+    - Lighter shade: Unamanged memory
+    - Even lighter shade: Unmanaged recent memory. 
+
+    - Grey: Spilled to disk memory 
+
+For a detailed explanation on the different types of memory refer to the Worker Memory management
+documentation `Worker Memory management documentation <https://distributed.dask.org/en/latest/worker.html#memory-management>`_
+
+Processing/CPU/Occupancy
+------------------------
+
+**Task Processing** 
+
+.. figure:: images/dashboard_task_processing.png
+    :alt: NEEDS ALT TEXT
+
+The Processing plot, shows the tasks being processed across each worker. Ideally you want, each worker to 
+process an equally amount of tasks. If one of the bars is completely white it means that 
+worker is idle and waiting for tasks. This usually happens when the computations are close to finish (nothing 
+to worry about), but it can also mean that the distribution of the task across workers is not optimized and you
+will want to revisit your code. 
+
+In this plot you can also find the colors red and green: **(PENDING)**
+
+- Green : 
+- Red :
+
+In this plot on the dashboard we have two extra tabs with the following information:
+
+**CPU Utilization**
+
+CPU usage per-worker (this needs some love, haven't found a nice way of describing this) 
+
+**Occupancy**
+
+This tab shows the occupancy, in time, per worker. The total occupancy for a worker is the total expected runtime
+for all tasks currently on a worker. For example, an occupancy of 10s means (it's an estimation) that it will take the 
+worker 10s to execute all the tasks it has currently been assigned.
+
+Task Stream
+-----------
+
+The task stream is a view of all the tasks across threads. Each row represents a thread and each rectangle represent 
+an individual tasks. The color for each rectangle corresponds to the kind of task being performed and it matches the color 
+of the Progress plot (see Progress section). This means that all the individual tasks of kind `inc` for example will have 
+the same randomly assigned color. 
```

Comment:
I have a few thoughts here 😃 

* This is the first place you've introduced the idea of threads. This is often a super confusing topic, especially in the context of distributed computing, where threads, processes, and workers are all overlapping but distinct concepts. I think you've done a great job overall on this page totally avoiding going down that rabbit hole but still providing some excellent information, but I worry that using "threads" here will distract from the real message of this section, which is about tasks. What if it says something like:
  > The task stream is a view of all the tasks the scheduler has assigned.

  Would that work?
* That said, I love your second sentence. It makes it super clear what the purpose of this graph is. Is there any way to avoid "thread" here but still be technically correct?
* > The color for each rectangle corresponds to the kind of task being performed

  It seems like the color corresponds to the _specific_ task, not the kind of task. If it were the kind of task, then all functions would be one color, arithmetic another color, etc. Based on the next sentence, it seems like _each_ function gets its own color. Is that correct?
* > and it matches the color of the Progress plot (see Progress section).
  
  I think it would be clearer to provide the detail of the colors in whichever section comes first and then backwards link from the later section. So in this case, rather than linking forward to the Progress section, link backwards from the Progress section to the Task stream section.
* Nit: `each rectangle represent an individual tasks.` -> `each rectangle represents an individual task.`